includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -465,6 +472,11 @@ public static function get_validation_error_sanitization( $error ) {
 			$forced = 'with_preview';
 		}
 
+		if ( isset( $_GET[ self::REJECT_ALL_VALIDATION_ERRORS_QUERY_VAR ] ) && AMP_Validation_Manager::has_cap() ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
As opposed to adding a new condition here, I suggest using the `amp_validation_error_sanitized` filter below, since these debug scenarios are outside the normal band of validation.

Reply:
Good idea, how does [this look]( https://github.com/ampproject/amp-wp/pull/3448/files#diff-ff6d70c0505290e11be8ca83967c2af0R285)?